web/custom-plugins/forecast-of-response/src/components/Operations/OperationsTable.tsx

```diff
@@ -13,43 +13,29 @@ const columns: Array<string> = [
 ]
 
 type TOperationRow = {
+  _id?: string
   name: string
-  start: number | string
-  end: number | string
+  start: Date | string
+  end: Date | string
   location: string
   creator: string
-  status:
-    | OperationStatus.DRAFT
+  status?:
     | OperationStatus.UPCOMING
-    | OperationStatus.ONGOING
+    | OperationStatus.IN_PROGRESS
     | OperationStatus.CONCLUDED
```

Comment:
Never seen this syntax before. Is it the same as writing "status?: OperationStatus"?